<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-114</y>
        <width>472</width>
        <height>424</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="mainLayout">
         <item>
          <layout class="QHBoxLayout" name="autoRotateLayout">
           <item>
            <widget class="QLabel" name="autoRotateLabel">
             <property name="text">
              <string>Auto Rotate</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="autoRotateCheckBox">
             <property name="text">
              <string notr="true">Enable</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="triggerValueLayout">
           <item>
            <widget class="QLabel" name="triggerValueLabel">
             <property name="text">
              <string>Trigger Value</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="triggerValueEdit">
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>200</number>
             </property>
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="triggerCommandLayout">
           <item>
            <widget class="QLabel" name="triggerCommandLabel">
             <property name="text">
              <string>Trigger Command</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="triggerCommandEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="triggerCountLayout">
           <item>
            <widget class="QLabel" name="triggerCountLabel">
             <property name="text">
              <string>Trigger Count</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="triggerCountEdit">
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="colorScaleSettingLayout">
           <item>
            <widget class="QLabel" name="colorScaleSettingLabel">
             <property name="text">
              <string>Colored Scale</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="colorScaleSettingCheckBox">
             <property name="text">
              <string>Enable</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="triggerThresholdLayout">
           <item>
            <widget class="QLabel" name="colorScaleThresholdsLabel">
             <property name="text">
              <string>Thresholds</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="triggerThresholdChooserLayout">
             <item>
              <widget class="QSpinBox" name="lowerThresholdSpinBox">
               <property name="maximum">
                <number>200</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="upperThresholdSpinBox">
               <property name="maximum">
                <number>200</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="colorScaleColorsLayout">
           <item>
            <widget class="QLabel" name="colorScaleColorsLabel">
             <property name="text">
              <string>Colors</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="colorScaleColorButtonsLayout">
             <item>
              <widget class="ColorChooserButton" name="lowerColorPushButton">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="ColorChooserButton" name="midColorPushButton">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="ColorChooserButton" name="upperColorPushButton">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="logDataSettingLayout">
           <item>
            <widget class="QLabel" name="logDataSettingLabel">
             <property name="text">
              <string>Log Data</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="logDataSettingCheckBox">
             <property name="text">
              <string>Enable</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="logDataFileNameLayout">
           <item>
            <widget class="QLabel" name="logDataFileNameLabel">
             <property name="text">
              <string>Log File Name</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLineEdit" name="logDataFileNameLineEdit"/>
             </item>
             <item>
              <widget class="QPushButton" name="logDataFileChooserPushButton">
               <property name="text">
                <string>Select</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="interpolationOptionsLayout">
           <item>
            <widget class="QLabel" name="interpolationLabel">
             <property name="text">
              <string>Interpolation Level</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="interpolationSpinBox">
             <property name="maximum">
              <number>10</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorChooserButton</class>
   <extends>QPushButton</extends>
   <header>colorchooserbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
